packages/playwright/src/generators/init/init.ts

```diff
@@ -70,4 +76,25 @@ export async function initGenerator(tree: Tree, options: InitGeneratorSchema) {
   return runTasksInSerial(...tasks);
 }
 
+function addPlugin(tree: Tree) {
+  const nxJson = readNxJson(tree);
+  nxJson.plugins ??= [];
+
+  if (
+    !nxJson.plugins.some((p) =>
+      typeof p === 'string'
+        ? p === '@nx/playwright/plugin'
+        : p.plugin === '@nx/playwright/plugin'
+    )
+  ) {
+    nxJson.plugins.push({
+      plugin: '@nx/playwright/plugin',
+      options: {
+        targetName: 'e2e',
```

Comment:
do we need the `ciTargetName` here as well?

Reply:
Not adding it to options as its not added in they cypress generator